  <launch>
  
    <node name="airsimnode" pkg="airsim" type="airsimnode.py" respawn="false" output="screen">

            <param name="rate" type="int" value="10" />
            <param name="imu_rate" type="int" value="100" />
            
            <param name="camera1_active" type="int" value="1" />
            <param name="camera2_active" type="int" value="0" />
            <param name="camera3_active" type="int" value="0" />
            <param name="lidar_active" type="int" value="0" />
            <param name="gpulidar_active" type="int" value="0" />
            <param name="echo_active" type="int" value="0" />
            <param name="imu_active" type="int" value="1" />
            <param name="pose_active" type="int" value="1" />
            <param name="carcontrol_active" type="int" value="1" />

            <param name="camera1_frame_id" type="string" value="base_camera" />
            <param name="lidar_frame_id" type="string" value="base_laser" />
            <param name="echo_frame_id" type="string" value="base_echo" />
            <param name="imu_frame_id" type="string" value="base_link" />
            <param name="pose_frame_id" type="string" value="world" />

            <param name="topic_scene_camera1" type="string" value="airsim/rgb/image" />
            <param name="topic_segmentation_camera1" type="string" value="airsim/segmentation/image" />
            <param name="topic_depth_camera1" type="string" value="airsim/depth/image" />
            <param name="topic_lidar" type="string" value="airsim/lidar" />
            <param name="topic_lidar_groundtruth" type="string" value="airsim/lidargroundtruth" />
            <param name="topic_echo" type="string" value="airsim/echo" />
            <param name="topic_imu" type="string" value="airsim/imu" />
            <param name="topic_pose" type="string" value="airsim/gtpose" />

            <param name="vehicle_name" type="string" value="airsimvehicle" />
            <param name="imu_name" type="string" value="imu" />
            <param name="lidar_name" type="string" value="lidar" />
            <param name="echo_name" type="string" value="echo" />
            <param name="camera1_name" type="string" value="front_center" />

            <param name="camera1_sceneonly" type="int" value="0" />
            <param name="camera1_mono" type="int" value="0" />            
            <param name="camera_height" type="int" value="540" />
            <param name="camera_width" type="int" value="960" />

            <param name="carcontrol_input_topic" type="string" value="cmd_vel" />
      </node>
      <node name="pose_to_odom" pkg="airsim" type="pose_to_odom.py" respawn="false" output="screen"/>
      <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <rosparam file="$(find airsim)/drive/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find airsim)/drive/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find airsim)/drive/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find airsim)/drive/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find airsim)/drive/base_local_planner_params.yaml" command="load" />
        
        <param name="controller_frequency " type="double" value="6.0" />
      </node>
      <node name="simple_goal" pkg="airsim" type="simple_goal.py" respawn="false" output="screen"/>
  </launch>